### ISSUE TITLE
Gamepad crazyness

### ISSUE THREAD

**Issue by ghorricks** [dimok789/loadiine_gx2#69]
Looks like the build from yesterday had an issue with gamepad left stick not being calibrated... in other words it was as if the left stick was being pushed LEFT(ish) within loadiine. WiiU 5.5.1 E

**Comment by dimok789** [comment]
Maschell can you take a look at this? It looks like an issue with your stuff maybe? Also a setting for that stuff would be great so it is not always applied.

**Comment by Maschell** [comment]
Do you have anything in the usb ports?

**Comment by ghorricks** [comment]
Thanks for the quick response. God thinking - Yes there is something in the USB!!! :)
A 3rd party GameCube USB adaptor:
https://www.amazon.co.uk/gp/product/B00UVX0588/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1

**Comment by ghorricks** [comment]
A GBAtemp user suggested trying a version of loadiine GX2 which seemed to work fine.
I got the link from : http://loadiine.ovh/:
https://mega.nz/#!ykVVFL5Y!ru3Ps7pk3Q--cfwzf1zdqWyzkgxaXF7eUH-JfyLEkZg

**Comment by Maschell** [comment]
Is a gamepad in the adapter?

**Comment by ghorricks** [comment]
A wavebird receiver (port 1) but the pad isn't on which I guess doesn't matter.

**Comment by Maschell** [comment]
Can you try to remove it from the port?

**Comment by ghorricks** [comment]
Okay will do - Just need to replace the working (old) version of loadiine with todays once I am infront of the WiiU. Gimme 1 hour.

**Comment by ghorricks** [comment]
Okay, here are my findings.
I reverted to using the latest build available (loadiine_gx2_8fc0ade) and verified that the USB GameCube adapter and wavebird receiver were in use and causing an issue with the gamepad left stick.

I then tried a cold boot, exploit and loadiine again without the wavebird adapter but WITH the USB GC.
It still affected the gamepad left stick.

Finally, I unplugged the USB, cold booted, exploit and loadiine... working fine.

So it looks like this 3rd party USB GC adapter causes the issue with the left stick on the gamepad.
